service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/RoboticsDataServiceTest.java

```diff
@@ -491,6 +492,42 @@ void shouldNotHaveCaseIdPropertyWhenRoboticsDataDeserializes() throws IOExceptio
         }
     }
 
+    @Nested
+    class FeePaid {
+        private CaseData.CaseDataBuilder caseDataBuilder;
+
+        @BeforeEach
+        void setup() {
+            caseDataBuilder = prepareCaseData().toBuilder();
+        }
+
+        @Test
+        void shouldGetFeePaidFromCaseData() throws JsonProcessingException {
+            caseDataBuilder.amountToPay("100000");
+
+            RoboticsData roboticsData = roboticsDataService.prepareRoboticsData(caseDataBuilder.build(), CASE_ID);
+            String returnedRoboticsJson = roboticsDataService.convertRoboticsDataToJson(roboticsData);
+
```

Comment:
do we need to convert to json here? Given we are just asserting result for `feePaid`, could we do 
```java
caseDataBuilder.amountToPay("100000");

RoboticsData roboticsData = roboticsDataService.prepareRoboticsData(caseDataBuilder.build(), CASE_ID);

assertThat(roboticsData.getFeePaid()).isEqualTo(1000.00);
```

Reply:
Agree. Changed it now.